core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -115,6 +120,10 @@ var ExplorationEditorTranslationTab = function() {
     by.css('.protractor-test-translation-solution-tab'));
 
   var contentTabText = element(by.css('.protractor-test-content-text'));
+  var audioOverFiveMinutesErrorMessageElement = element(by.className(
+    'oppia-audio-file-upload-field-error-message'));
+  var uploadAudioButton = element.all(
+    by.className('protractor-test-upload-audio-button')).last();
```

Comment:
When an audio file is already uploaded, and we try to replace it, at that time there exist two elements with this class. :)
For more reference, you can try locating the element in a situation where an audio translation is already uploaded for a card.
Thanks.

Reply:
can we have two different protractor-test class for these buttons?

Reply:
@DubeySandeep We can. But I believe it is better to deal with it this way. Since the uploadAudioButton belongs to ExplorationEditorTranslationTab.js, it will become complex to use it in uploadAudio function, as the function uploadAudio would then have to determine whether it should use the former upload audio button element or the latter.(Though it is very easy for us to understand and implement, I think it will become quite complex and hard to understand for others).
In the current usage, the uploadAudioButton contains the upload audio button displayed on the page. So, I think it is better this way.
What are your views on this?